app/models/concerns/sufia/solr_document_behavior.rb

```diff
@@ -7,6 +7,14 @@ module SolrDocumentBehavior
     include Sufia::SolrDocument::Export
     include Sufia::SolrDocument::Characterization
 
+    def all_titles
+      Array.wrap(self[Solrizer.solr_name('title')])
+    end
+
+    def all_descriptions
```

Comment:
Yes. Same as above. Since I only focused on works I renamed it from `description` because it broke much.